docs/index.md

```diff
@@ -46,6 +46,20 @@ gitStream is a workflow automation tool that enables you to use YAML configurati
 
 		gitStream can be installed for one repo, specific repos, or all repos in your organization. We recommend installing for **all** because it will ensure all new repos are able to use gitStream. You can change this setting at any time in the future.
 
+=== ":fontawesome-brands-github: GitHub Server"
+
+	1. Create and install a [Custom GitHub App for Self-Hosted GitHub Server](/custom-github-app)
+	2. <a href="https://app.linearb.io/login" target="_blank">Login</a> to the LinearB app.
+		1. Make sure the GHES is integrated with LinearB
```

Comment:
if i know nothing, then it would help if it says where to find it (its in the Git company settings), or point to the help docs